docs/release-notes/1291-improve-wait-page-loading

```diff
@@ -0,0 +1,5 @@
+:orphan:
+
+**Improvements**
+
+- Improve wait page loading and speed up TensorBoard and Notebook open up.
```

Comment:
Improve wait page loading and optimize startup for new TensorBoard and Notebook tasks.